src/software/ai/hl/stp/action/stop_action.h

```diff
@@ -27,15 +27,13 @@ class StopAction : public Action
     StopAction() = delete;
 
     /**
-     * Updates the params that cannot be derived from the world for this action
+     * Updates the params that can be derived from the world for this action
      *
-     * @param robot the robot that should stop
-     * @param coast Whether or not to coast to a stop. If this is false, the robot will
-     * try actively brake to come to a stop
+     * @param ball The world to update from
      */
```

Comment:
This comment is outdated, you can remove it

Reply:
thanks, fixed :)